hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -632,6 +632,10 @@ public void createBucket(
     StorageType storageType = bucketArgs.getStorageType() == null ?
         StorageType.DEFAULT : bucketArgs.getStorageType();
     BucketLayout bucketLayout = bucketArgs.getBucketLayout();
+    if (bucketLayout == null) {
+      bucketLayout = BucketLayout
+          .fromString(conf.get(OMConfigKeys.OZONE_DEFAULT_BUCKET_LAYOUT));
```

Comment:
OMConfigKeys.OZONE_DEFAULT_BUCKET_LAYOUT is basically a OM server config and we are using it in client code but I guess we should be safe as OMConfigKeys is defined in the `common` module?

Reply:
@sadanand48 Thanks for reviewing this PR. It seems safe to me. The logs are reading the bucket layout from the client and the client in return reads it from the command args. I don't see another approach to this.